dds_web/templates/mail/project_release.html

```diff
@@ -10,11 +10,13 @@
   </tr>
   <tr>
     <td style=" font-family: arial, sans-serif; font-size: 14px; vertical-align: top;">
-      <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: bold; margin: 0; Margin-bottom: 5px;">The
-        project {{project_id}} is now available for your access in the SciLifeLab Data Delivery System (DDS).</p>
-      <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 10px;">
-        The DDS is a system for SciLifeLab infrastructures to deliver data to researchers in a fast, secure and simple
-        way.</p>
+      <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 5px;">
+        The following project is now available for your access in the SciLifeLab Data Delivery System (DDS) and you can now download your data.
```

Comment:
Remove the second `now`
```suggestion
        The following project is now available for your access in the SciLifeLab Data Delivery System (DDS) and you can download your data.
```